fred2/briefingeditordlg.cpp

```diff
@@ -473,6 +475,9 @@ void briefing_editor_dlg::update_data(int update)
 			}
 			strcpy_s(ptr->icons[m_last_icon].closeup_label, buf);
 
+			if (m_icon_scale)
```

Comment:
You sure this is correct?
m_icon_scale is set to -1 by default, which evaluates to true (as only 0 is false)

Reply:
It defaults to -1, but any new icon is actually set to 100 (percent). But either way, we shouldn't allow negative values because that's nonsensical and I'm not sure what the icon drawing function will actually do with it. Changed the check to > 0.